packages/iota/src/validation/unlocks/unlocks.ts

```diff
@@ -25,9 +25,9 @@ export function validateUnlocks(unlocks: UnlockTypes[]): IValidationResult {
         switch (unlock.type) {
             case SIGNATURE_UNLOCK_TYPE:
                 if (seenSignatures.some(sig =>
-                        sig.type === unlock.signature.type &&
-                        sig.publicKey === unlock.signature.publicKey &&
-                        sig.signature === unlock.signature.signature)
+                    sig.type === unlock.signature.type &&
```

Comment:
This formatting hurts my eyes.
Do you agree that this is much more digestible regarding the "nesting" (and wtf is going on in the condition) ?
```
if (
    seenSignatures.some(sig =>
                        sig.type === unlock.signature.type &&
                        sig.publicKey === unlock.signature.publicKey &&
                        sig.signature === unlock.signature.signature
                       )
) {
```